Description/en-US.md

```diff
@@ -1,75 +1,67 @@
 # English (US)
+<!-- Since EN-US is supposed to be the "default" description, omit this header when pasting to distribution platforms -->
 
-A Fabric-based modpack for Minecraft that focuses on performance and graphics optimizations. 
+A Fabric-based Minecraft modpack focusing on performance and graphics enhancements.
 
-[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
+[<img src="https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg" alt="Current review" height="140"/> - a YouTube showcase by Rexus](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
```

Comment:
Usually markdown won't render around HTML, so you might need to convert this line to HTML in its entirety. May also bold the subtitle?

Reply:
it would be nice to bold it, actually. How do I actually convert this to HTML (this really just depends on how MR and CF would interpret it)?

Reply:
Something like:
```html
<a href=https://www.youtube.com/watch?v=bb8G9X5Q_4I><img src="https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg" alt="Current review" height="140"/> - A YouTube showcase by Rexus</a>
```